uktrade/data-hub-leeloo
# GVA Multiplier Admin

### Description of change
Initial GVA Multiplier admin page. Allows the multiplier value only to be updated. When it is updated it will in turn update all related investment projects.

### Design decision
 Used a task to update all the projects this is my preferred method but if there is alternative that is preferred don't mind updating.

### To test
- Make sure you restart celery if you are not ALWAYS_EAGER
- Go to http://localhost:8000/admin/investment/gvamultiplier/
- Test filtering and search 
- Update a project, select a sector and a foreign equity amount. 
- View http://localhost:8000/v3/investment/<uuid> and check gross value added is set
-  In the admin find the multiplier for the sector and try and delete (you should be prevented from doing so)
- Update the multiplier value and then check the project has been updated.

### Checklist
